.env

```
@@ -0,0 +1,18 @@
+TYPO3_CONF_VARS__BE__debug='true'
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The TYPO3_CONF_VARS__BE__debug key should be in uppercase

(LowercaseKey)